app/assets/stylesheets/welcome.scss

```diff
@@ -1,3 +1,12 @@
 // Place all the styles related to the Welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
```

Comment:
It's a good practice to use classes as selectors instead of raw tags (even for trivial cases like this).